libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -257,6 +257,9 @@ class SRV_Channel {
     // high point of angle or range output
     uint16_t high_out;
 
+    // override loop counter
+    uint16_t override_counter;
```

Comment:
make this an array in the SRV_Channels class, not a var in the individual channel